scripts/scil_score_tractogram.py

```diff
@@ -149,6 +160,11 @@ def _build_arg_parser():
                    help="Dilate inclusion masks n-times. Default: 0.")
     g.add_argument("--remove_invalid", action="store_true",
                    help="Remove invalid streamlines before scoring.")
+    g.add_argument('--min_length', type=float, default=10.,
+                   metavar='m',
```

Comment:
Were you aware that length can already be included in the config file? It can even be different for each bundle! See description above.

Reply:
Not sure why I added it. I removed it in 589c7b6